### DISCUSSION TITLE
[Suggestion] Night light | redshift

### DISCUSSION THREAD

**Discussion by SirDagen** [linuxmint/discussions#723] [Ideas]
Dear people of Cinnamon,

Thank you so much for creating this wonderful Cinnamon Desktop. I really love it. 

I have seen that there is now ‘night light’ as an alternative to ‘redshift’. 

I would be very happy about the following extension: With redshift I can not only set a night temperature, but also a day temperature (e.g. 2800K and 3800K). I enjoy this a lot, because laptop screens are often far too blue by default and it was always an easy fix for me, since my eyes do not work well under too much blue light. 

e.g. redshift -l 49.54:5.73 -t 3800:2800

ALSO: redshift continously shifts between the two values (and not abruptly), which is another big advantage.

(P.S. No idea, if it would be easy to add a gamma setting as well.)

**Comment by Jopp-gh** [comment]
There are better alternatives than Redshift, which is a very old and outdated software.

Check out [my free scripts](https://buymeacoffee.com/jopp.gh/e/353734), to change monitor brightness and redness.

**Reply by FlowXP** [reply]
also would be usefull to see somewhere the temperature number whne sliding
![image](https://github.com/user-attachments/assets/bff02e88-997e-4513-a44d-c007b2f5b86a)

**Reply by Jopp-gh** [reply]
@FlowXP 
So far, I didn't upgrade to Wilma. If I'm right, night light was introduced with Wilma -try to upgrade to Xia, maybe you've luck and night light got some more features.

**Comment by SirDagen** [comment]
Okay. I dont know what to do with (your) scripts. I prefer that it is all automatic and build into the system. Redshift currently does all I want it to do.

**Reply by Jopp-gh** [reply]
I didn't push anybody to use my scripts. @SirDagen 
Why giving me a thumbs down? Is my comment offending to you?

**Comment by Matthias-Hermsdorf** [comment]
Setting the day temperature as well would be nice.

The automatic timeplan redshifts the screen when the sun is down. At this time my room could be bright. Could you extend the nightlight to get triggered by the screen brightness or if present by an ambient sensor. So to have 2500k at 10% screen brightness and 5500k at 100%?

**Reply by SirDagen** [reply]
Nice idea. - In the evening and at night, I specifically use reddish and darker light so as not to upset my body rhythm. So I dont need it (and I wouldn't want it), but having the option would be nice.

**Comment by SirDagen** [comment]
I'd like to say that I still really cannot live without redshift as the current night light option isn't powerful enough for me to change the colour of my screen during day, night and twilight.

**Reply by ghost** [reply]
Yes that would be a nice addition, Gammashift does it as well.
